demos/src/demo.scss

```diff
@@ -13,11 +17,19 @@ $o-loading-is-silent: false;
 }
 
 .demo-card--light {
-	background: #f6e9d8;
+	@if (global-variable-exists(o-brand)) {
+		background: if($o-brand == 'internal', white, #fff1e5);
+	} @else {
+		background: #fff1e5;
+	}
 }
 
 .demo-card--dark {
-	background: #333333;
+	@if (global-variable-exists(o-brand)) {
+		background: if($o-brand == 'internal', #262a33, #333333);
```

Comment:
In other demos lots of our dark/inverse backgrounds for master are slate. Can't this be slate for all cases?

Reply:
That's just was it was, I don't really mind either way.